transform.ts

```diff
@@ -229,14 +230,20 @@ function createInterface(path: NodePath, componentTypes: CollectedTypes) {
 
   // Add the TS types before the function/class
   getFunctionParent(path).insertBefore(
-    j.tsInterfaceDeclaration(
-      j.identifier(typeName),
-      j.tsInterfaceBody(types.types.map(createPropertySignature))
-    )
+    options.declarationStyle === "type"
```

Comment:
We only check if the value is `type`. In this way, in case of `interface` or   unaccepted value the default will be to output interface.
LMK if this behavior is ok or if you'd rather have a throw or else in case of not listed declaration style value

Reply:
This is perfect!